code/game/objects/items/weapons/holosign.dm

```diff
@@ -16,6 +16,23 @@
 	var/creation_time = 0 //time to create a holosign in deciseconds.
 	var/holosign_type = /obj/structure/holosign/wetsign
 	var/holocreator_busy = FALSE //to prevent placing multiple holo barriers at once
+	var/wet_timer = FALSE
+
+/obj/item/holosign_creator/AltClick(mob/user)
+	if(!holosign_type == /obj/structure/holosign/wetsign)
+		.=..()
+		return
+	wet_timer = !wet_timer
+	switch(wet_timer)
+		if(TRUE)
+			to_chat(user, "<span class='notice'>You enable the W.E.T.(wet evaporation timer)\nAny newly placed holographic signs will clear after the likely time it takes for a mopped tile to dry.</span>")
+		if(FALSE)
+			to_chat(user, "<span class='notice'>You disable the W.E.T.(wet evaporation timer)\nAny newly placed holographic signs will now stay indefinitely.</span>")
+
+/obj/item/holosign_creator/examine(mob/user)
+	if(ishuman(user) && holosign_type == /obj/structure/holosign/wetsign)
+		desc += "\n<span class='info'>Alt Click to [wet_timer ? "activate" : "deactivate" ] its built-in wet evaporation timer.</span>"
+	. = ..()
```

Comment:
instead of doing this, make the janitor holoprojector its own subtype, and override the procs from there.